mlprimitives/jsons/sklearn.linear_model.Lasso.json

```diff
@@ -0,0 +1,129 @@
+                "types": {
+                    "int":{
+                        "description": "random_state is the seed used by the random number generator"
+                    },
+                    "RandomState":{
+                        "description": "If an array is passed, penalties are assumed to be specific to the targets. Hence they must correspond in number."
+                    },
+                    "NoneType": {
+                        "description": "If none, the random number generator is the RandomState instance used by np.random"
+                    }
+                },
+                "default": null
+            },
+            "precompute": {
+                "type": "multitype",
+                "description": "Whether to use a precomputed Gram matrix to speed up calculations. Gram natrix can also be passed as argument.",
+                "types":{
+                    "bool":{
+                        "description": "whether to use the precomputed gram matrix"
+                    },
+                    "array":{
+                        "description": "Gram matrix to speed up calculations"
+                    }
+                },
+                "default": false
+            },
+            "selection": {
+                "type": "str",
+                "description": "If set to ‘random’, a random coefficient is updated every iteration rather than looping over features sequentially by default.",
+                "values": ["cyclic", "random"],
+                "default": "cyclic"
+            }
+        },
+        "tunable": {
+            "normalize": {
+                "type": "bool",
+                "default": false,
+                "description": "This parameter is ignored when fit_intercept is set to False. If True, the regressors X will be normalized before regression by subtracting the mean and dividing by the l2-norm."
+            },
+            "fit_intercept": {
+                "type": "bool",
+                "default": true,
+                "description" : "Whether to calculate the intercept for this model. If set to False, no intercept will be used in calculations (e.g. data is expected to be already centered)."
+            },
+            "alpha": {
+                "type": "multitype",
+                "description": "Regularization strength. Alpha corresponds to C^-1 in other linear models such as LogisticRegression or LinearSVC.",
+                "types": {
+                    "float":{
+                        "description": "Must be a positive float. Larger values specify stronger regularization"
+                    },
+                    "array":{
+                        "description": "If an array is passed, penalties are assumed to be specific to the targets. Hence they must correspond in number."
+                    }
+                },
+                "default": 1.0
+            },
+            "tol": {
+                "type": "float",
```

Comment:
Tunable hyperparameters need to always have either a `range` (numerical) or a `values` (categorical) entry. The only exception is the `bool` type, sine the possible values are known in advance.

So, can you add a `range` entry here with the minimum and maximum "reasonable" values for this hyperparameter?